src/ngraph/frontend/onnx_import/op/quantize_linear.cpp

```diff
@@ -0,0 +1,99 @@
+//*****************************************************************************
+// Copyright 2017-2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <cstddef>
+#include <cstdint>
+#include <iterator>
+#include <memory>
+
+#include "exceptions.hpp"
+#include "ngraph/axis_set.hpp"
+#include "ngraph/op/quantize.hpp"
+#include "ngraph/shape.hpp"
+#include "quantize_linear.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace set_1
+            {
+                ngraph::NodeVector quantize_linear(const ngraph::onnx_import::Node& node)
+                {
+                    ngraph::NodeVector inputs{node.get_ng_inputs()};
+                    std::shared_ptr<ngraph::Node> x = inputs.at(0);
+                    std::shared_ptr<ngraph::Node> y_scale = inputs.at(1);
+                    std::shared_ptr<ngraph::Node> y_zero_point = inputs.at(2);
+
+                    bool has_axis = false;
+                    ngraph::Shape x_shape = x->get_shape();
```

Comment:
Similarly here, no need to use `ngraph::` namespace. `Shape` and `AxisSet` object are unique in `ngraph` namespace we're already in.